src/qibolab/_core/instruments/qm/components/configs.py

```diff
@@ -22,6 +23,27 @@
 
 DEFAULT_SAMPLING_RATE = 1e9
 
+DEFAULT_FEEDFORWARD_MAX = 2 - 2**-16
+"""Maximum feedforward tap value"""
+DEFAULT_FEEDBACK_MAX = 1 - 2**-20
+"""Maximum feedback tap value"""
+
+
+def normalize_feedforward(taps: list[float], threshold: float):
+    max_value = np.max(np.abs(taps))
+    if max_value > threshold:
+        return (threshold * np.array(taps / max_value)).tolist()
+    else:
+        return taps
+
+
+def normalize_feedback(taps: list[float], threshold: float):
+    new_taps = np.array(taps)
+    if np.any(np.abs(taps) > threshold):
+        new_taps[new_taps > threshold] = threshold
+        new_taps[new_taps < -threshold] = -threshold
+    return new_taps.tolist()
```

Comment:
Other than simplifying the function, it would be even better to provide a docstring, motivating why these functions are required (if relevant, citing the QM docs).